priv/static/specs/http_api_spec.yaml

```diff
@@ -79,3 +134,59 @@ definitions:
     properties:
       version:
         type: "string"
+  SessionsList:
+    type: "object"
+    properties:
+      TotalCount:
+        type: integer
+      Sessions:
+        type: array
+        items:
+          $ref: '#/definitions/SessionInfo'
+  SessionInfo:
+    type: "object"
+    properties:
+      IMSI:
+        type: string
+      IMEI:
+        type: string
+      MSIDN:
+        type: string
+      Version:
+        type: string
+      RemoteControlIp:
+        type: string
+      RemoteDataIp:
+        type: string
+      ControlPort:
```

Comment:
this leaks to much internal detail

@mgumz please have a look also.

My idea would be make this seomthing like

```
LocalGTPentity:
  GTP-C:
   Version: {v1,v2}
    TEID: ....
    IP: ...
  GTP-U:
   Version: v1
    TEID: ....
    IP: ...
RemoteGTPentity:
  GTP-C:
   Version: {v1,v2}
    TEID: ....
    IP: ...
  GTP-U:
   Version: v1
    TEID: ....
    IP: ...
```

Reply:
@RoadRunnr LocalGTPEntity ... :)